src/Frost/Effects/Git.hs

```diff
@@ -0,0 +1,26 @@
+{-# LANGUAGE TemplateHaskell #-}
+module Frost.Effects.Git where
+
+import Data.String.Utils
+import Data.List
+import Polysemy
+import SimpleCmd.Git
+
+import qualified Data.Text as T
+
+data GitEffect m a where
+  DevsList :: GitEffect m [String]
+
+makeSem ''GitEffect
+
+runGitEffect :: Member (Lift IO) r => Sem (GitEffect ': r) a -> Sem r a
+runGitEffect = interpret $ \case
+  DevsList -> sendM getContributors
+
+getContributors :: IO [String]
+getContributors  = do
+    output <- git "log" ["--pretty=short", "-s"]
+    let s = T.split (=='\n') (T.pack output)
+    let r = nub $ filter (\x -> startswith  "Author:" (T.unpack x)) s
```

Comment:
when dealing with lambdas, it's almost always more readable to go point free I think
```
filter (startswith "Author:" . T.unpack) s
``` 
but it's just a matter of style I gues

Reply:
I've used that converter for a long time to get a point free intuition http://pointfree.io/. Once you train your brain, you want to point free all the things